src/into_dart.rs

```diff
@@ -296,6 +303,35 @@ where
 
 impl<T> IntoDartExceptPrimitive for Vec<T> where T: IntoDartExceptPrimitive {}
 
+impl<T> IntoDart for HashSet<T>
+where
+    T: IntoDartExceptPrimitive,
+{
+    fn into_dart(self) -> DartCObject {
+        self.into_iter().collect::<Vec<_>>().into_dart()
+    }
+}
+
+impl<T> IntoDartExceptPrimitive for HashSet<T> where T: IntoDartExceptPrimitive {}
+
+impl<K, V> IntoDart for HashMap<K, V>
+where
+    K: IntoDart,
+    V: IntoDart,
+    (K, V): IntoDartExceptPrimitive,
+{
+    fn into_dart(self) -> DartCObject {
+        self.into_iter().collect::<Vec<_>>().into_dart()
+    }
+}
```

Comment:
We are sending these values as tuples in case of HashMap? Can we add small comment or something to the docs how that will look like in the dart side? Is it a `List<dynamic>`?

Reply:
Yes, as tuples, and `List<dynamic>`